wpilib/wpilib/robotdrive.py

```diff
@@ -25,10 +25,9 @@ def _freeRobotDrive(allocatedSpeedControllers):
         if hasattr(sc, "free"):
             sc.free()
 
-@DeprecationWarning
 class RobotDrive(MotorSafety):
     """
-    Deprecated. Use drive/* instead.
+    .. deprecated:: 2018.0.0. Use DifferentialDrive, KilloughDrive, or MecanumDrive instead.
```

Comment:
`. Use DifferentialDrive, KilloughDrive, or MecanumDrive instead.` isn't part of the version number.